### ISSUE TITLE
Shebangs?

### ISSUE THREAD

**Issue by IcyRamen** [geekcomputers/Python#44]
Shouldn't all the .py files all have shebangs?
For example:
#!/usr/bin/python3
(Assuming the user is on Linux/Unix. If the user is on windows it will be read as a comment and no harm will be done.)

**Comment by teror4uks** [comment]
but is it necessary? i can use py2 and this coment in first line will be prevent. 
P.S. It is my opinion

**Comment by IcyRamen** [comment]
Well, it ensures that the file can be run with a double click from a file explorer instead of the need to run  `python2 filename.py` from the terminal.

**Comment by himanshuraja** [comment]
yeah and Pyinstaller can do also,but wondering if the suggestions were in the code in form of comments for help

**Comment by nix-py** [comment]
I think all scripts should have this shebang
`#!/usr/bin/env python`

**Comment by geekcomputers** [comment]
Thank you for you contribution to the repo and the community
